loadsave.c

```diff
@@ -132,7 +151,20 @@ LOAD()
 		RAM[STATUS] = 0;
 		a = 0;
 	} else {
-		FILE *f = fopen(filename, "rb");
+        
+        //use correct file location for ios
+        //use correct file location for ios
+        #if __APPLE__ && (TARGET_IPHONE_SIMULATOR || TARGET_OS_IPHONE)
+        char fullFileName[256];
+        //concatenating the path string returned from HOME
+        strcpy(fullFileName,getenv("HOME"));
```

Comment:
coding style: space after comma, please, to be consistent with the surrounding code.